.travis.yml

```diff
@@ -0,0 +1,7 @@
+language: node_js
+node_js:
+  - "4"
+before_script:
+  - npm install -g istanbul
```

Comment:
I think you need more than only istanbul before running srcipt. Just put 'npm install' to be on the safer side